testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -1584,6 +1585,43 @@ private Optional<ZonedDateTime> getMaterializedViewLastFreshTime(String material
         return Optional.ofNullable(lastFreshTime);
     }
 
+    @Test
+    public void testColumnCommentMaterializedView()
+    {
+        if (!hasBehavior(SUPPORTS_COMMENT_ON_MATERIALIZED_VIEW)) {
+            if (hasBehavior(SUPPORTS_CREATE_MATERIALIZED_VIEW)) {
+                String viewName = "test_materialized_view_" + randomNameSuffix();
+                assertUpdate("CREATE MATERIALIZED VIEW " + viewName + " AS SELECT * FROM nation");
+                assertQueryFails("COMMENT ON COLUMN " + viewName + ".regionkey IS 'new region key comment'", "This connector does not support setting materialized view column comments");
+                assertUpdate("DROP MATERIALIZED VIEW " + viewName);
+                return;
+            }
+            throw new SkipException("Skipping as connector does not support MATERIALIZED VIEW COLUMN COMMENT");
+        }
+
+        String catalog = getSession().getCatalog().orElseThrow();
+        String schema = getSession().getSchema().orElseThrow();
+
+        QualifiedObjectName view = new QualifiedObjectName(catalog, schema, "test_materialized_view_" + randomNameSuffix());
+
+        createTestingMaterializedView(view, Optional.empty());
```

Comment:
Using `createTestingMaterializedView` looks redundant for me. How about directly using `assertUpdate`?

Reply:
refactored